src/Components/getList.js

```diff
@@ -1,29 +1,57 @@
 import React from 'react';
 import { FirestoreCollection } from 'react-firestore';
 import { getLocalToken } from '../lib/token.js';
+import { useHistory } from 'react-router-dom';
 
-class GetList extends React.Component {
-  render() {
-    return (
-      <FirestoreCollection
-        path={`${getLocalToken()}/`}
-        render={({ isLoading, data }) => {
-          return isLoading ? (
-            <div>loading...</div>
-          ) : (
-            <div>
-              <h1>Groceries</h1>
-              <ul>
-                {data.map(item => (
-                  <li key={item.id}>{item.itemName}</li>
-                ))}
-              </ul>
-            </div>
-          );
-        }}
-      />
-    );
+function EmptyList() {
+  let history = useHistory();
+
+  function handleClick() {
+    history.push('/add-item');
   }
+
+  return (
+    <div className="bg-white shadow pa2">
+      <h1 className="b f1">Welcome to your shopping list!</h1>
+      <p>
+        To get started add an item to your shopping list by clicking "add item
+        below"
+      </p>
+      <button onClick={handleClick} className="bg-green ph2 pv1 white f2 b">
+        Add an item
+      </button>
+    </div>
+  );
+}
+
+function FullList(props) {
+  return (
+    <div>
+      <h1>Groceries</h1>
+      <ul>
+        {props.data.map(item => (
+          <li key={item.id}>{item.itemName}</li>
+        ))}
+      </ul>
+    </div>
+  );
+}
+
+function GetList() {
+  return (
+    <FirestoreCollection
+      path={`${getLocalToken()}/`}
+      render={({ isLoading, data }) => {
+        return isLoading ? (
+          <div>loading...</div>
+        ) : data.length === 0 ? (
```

Comment:
I like that you seem to have a handle on react hooks, it's definitely cleaner, we're still learning them so we'll likely try to refactor this soon.  We might have to check in with you about merging since you're using hooks and we're handling state changes in the functionality we would move to your FullList function.  _weirdly, I had deja vu typing that even though I didn't know about hook until now_

Reply:
I was thinking the same thing! I don't think it should be too bad yes we should definitely sync up before merging.

Reply:
I second both of your all's comments! I think we should get together to refactor the code to include hooks, but kudos to you @nabilkaz for getting on that!